packages/node-core/src/indexer/storeCache/storeCache.service.ts

```diff
@@ -98,7 +91,8 @@ export class StoreCacheService {
   }
 
   isFlushable(): boolean {
```

Comment:
Maybe we need a force flush? I think with unfinalized/rewind/dynamic-ds and startup we might want to force flush